turbine/src/broadcast_stage/standard_broadcast_run.rs

```diff
@@ -506,8 +506,14 @@ impl BroadcastRun for StandardBroadcastRun {
     }
 }
 
-fn should_chain_merkle_shreds(_slot: Slot, cluster_type: ClusterType) -> bool {
-    cluster_type == ClusterType::Development
+fn should_chain_merkle_shreds(slot: Slot, cluster_type: ClusterType) -> bool {
+    match cluster_type {
+        ClusterType::Development => true,
+        ClusterType::Devnet => false,
+        ClusterType::MainnetBeta => false,
+        // Roll out chained Merkle shreds to ~5% of testnet.
+        ClusterType::Testnet => slot % 19 == 1,
```

Comment:
20 should be 5%, is 19 to ensure that it's not always the same leader slot index (1 to 4)?

Reply:
yeah, I picked some prime number to avoid those kind of frequencies.